lambdas/initialise-ipv-session/src/main/java/uk/gov/di/ipv/core/initialiseipvsession/InitialiseIpvSessionHandler.java

```diff
@@ -202,6 +208,19 @@ public APIGatewayProxyResponseEvent handleRequest(
                                     getJarUserInfo(claimsSet).map(JarUserInfo::evcsAccessToken),
                                     claimsSet));
 
+            if (configService.enabled(AIS_ENABLED)) {
```

Comment:
I've kept this in `InitialiseIpvSession` as the flag is used in audit events raised here.

Reply:
```suggestion
            if (!isReverification && configService.enabled(AIS_ENABLED)) {
```
If reverification, can we skip over all of this logic?

Reply:
I considered that, but when we turn the flag on we don't want the behaviour to be influenced by the orch request, otherwise when that gets turned off our behaviour may change.

Reply:
I guess, though with isReverification the clientOAuthSession updates would be redundant & a change from what we had originally